xcm/pallet-xcm/src/tests.rs

```diff
@@ -0,0 +1,180 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+use crate::mock::*;
+use frame_support::{assert_noop, assert_ok, traits::Currency};
+use polkadot_parachain::primitives::{AccountIdConversion, Id as ParaId};
+use xcm::{
+	opaque::v0::prelude::*,
+	v0::{Junction, Xcm},
+};
+
+#[test]
+fn send_reserve_asset_deposit_works() {
```

Comment:
should just be called `send_works`. I dont see anything specific about reserve asset deposit happening here